model/mobile_metric.go

```diff
@@ -0,0 +1,56 @@
+package model
+
+import (
+	"encoding/json"
+	"io"
+	"time"
+)
+
+type DeviceDetails struct {
+	Country        string `json:"country"`
+	APILevel       string `json:"api_level"`
+	BuildNumber    string `json:"build_number"`
+	BundleId       string `json:"bundle_id"`
+	DeviceId       string `json:"device_id"`
```

Comment:
Is there any field to identify whether its Android or IOS device?

Reply:
Currently we dont have. But with model name we can differentiate.